projects/canopy/src/lib/icon/icons.interface.ts

```diff
@@ -3,1039 +3,1039 @@
 
 export const lgIconAdd: Icon = {
   name: 'add',
-  data: `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32"><defs><path id="a" d="M17 15h8a1 1 0 010 2h-8v8a1 1 0 01-2 0v-8H7a1 1 0 010-2h8V7a1 1 0 012 0v8z"/></defs><use fill-rule="evenodd" xlink:href="#a"/></svg>`,
+  data: `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32"><defs><path id="a" d="M17 15h8a1 1 0 010 2h-8v8a1 1 0 01-2 0v-8H7a1 1 0 010-2h8V7a1 1 0 012 0v8z"/></defs><use fill-rule="evenodd" xlink:href="#a"/></svg>`
```

Comment:
Hmmm seems odd that the trailing comma has bene removed from all these lines. Prettier is supposed to add the comma's when they're missing on the pre-commit hook 🤔   I'll Slack you and see if there's something in your setup that's doing this

Reply:
I think this is due to the way the icons are setup. These interface files are actually generated on the pre-commit hook and so might not be setup to also work with prettier. Although I think they do have to be manually added to the commit so I wonder why it didn't run on them then. Could be you just need to run `npm install` locally as I think the trailing comma rule was added recently (cc @elenagarrone )

Reply:
Had a look with Owen and fixed the prettier version on my machine.

Reply:
Alex was somehow using an older version of Prettier, despite doing an `npm install`. Updating it and running it again formatted the files correctly 👍